### PR TITLE
Add Exomiser tutorial

### PR DESCRIPTION
**Pull request by yaseminbridges**
Added the `exomiser-tutorial.md` ready for the 04/04/2023 Exomiser tutorial. The tutorial has been added to the table of contents and the link the the .md file has also been linked in the `monarch-obo-training.md`

### CODE REVIEW

File: docs/courses/monarch-obo-training.md [hunk 1]
```diff
@@ -47,36 +48,36 @@ _Note: this is tentative and subject to change_
...
+| 2021/08/24 | [Contributing to OBO ontologies 2](../lesson/contributing-to-obo-ontologies.md)                                                                             |                                                                                                                                                                                                                                                                                                                      | [Here](https://www.youtube.com/watch?v=kAw4N2tYyco)                  |
+| 2021/08/17 | [Contributing to OBO ontologies](../lesson/contributing-to-obo-ontologies.md)                                                                               |                                                                                                                                                                                                                                                                                                                      |
```
Comment (matentzn) [inline docs/courses/monarch-obo-training.md:80]: Why the big diff?

File: docs/tutorial/exomiser-tutorial.md [hunk 2]
```diff
@@ -0,0 +1,680 @@
+## Tutorial: an introduction to Exomiser
```
Comment (matentzn) [inline docs/tutorial/exomiser-tutorial.md:1]: If you like you can include a [maturity indicator badge](https://oboacademy.github.io/obook/reference/obook-maturity-indicator/), see for example

https://oboacademy.github.io/obook/lesson/contributing-to-obo-ontologies/

**Review by matentzn — APPROVED** [review]
I made two comments but this looks truly awesome. Thank you ALL!!

File: docs/tutorial/exomiser-tutorial.md (hunk 2 above)
Comment (yaseminbridges) [reply docs/tutorial/exomiser-tutorial.md:1]: Yes, this will be perfect, I'll add it!

**Pull request merged**